### PR TITLE
PRIME-1041 business events on site + fix note/documents

### PR DESCRIPTION
**Pull request by hogmann**
Wrapped all the site components with the enrollee container causing some interesting behavior when you went to a site note or document page.

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-documents/adjudicator-documents.component.html [hunk 1]
```diff
@@ -1,75 +1,65 @@
-<app-adjudication-container [content]="content">
+<div class="">
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-documents/adjudicator-documents.component.html:1]: Unlike React you don't need a parent element in the template

Comment (hogmann) [reply prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-documents/adjudicator-documents.component.html:1]: Can't stop won't stop

File: prime-dotnet-webapi/Controllers/SitesController.cs [hunk 2]
```diff
@@ -719,5 +719,31 @@ public async Task<ActionResult<IEnumerable<Site>>> GetSitesByRemoteUserInfo(List
...
+        /// Gets all of the site registration notes for a specific site.
+        /// </summary>
+        /// <param name="siteId"></param>
+        /// <param name="businessEventTypeCodes"></param>
+        [HttpGet("{siteId}/events", Name = nameof(GetSiteBusinessEvents))]
+        [Authorize(Policy = AuthConstants.READONLY_ADMIN_POLICY)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(typeof(ApiResultResponse<IEnumerable<Status>>), StatusCodes.Status200OK)]
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Controllers/SitesController.cs:734]: Response Type here should be IEnumerable<BusinessEvent>

**Review by mtpultz — APPROVED** [review]
image

**Pull request merged**